q2_taxa/_visualizer.py

```diff
@@ -24,6 +24,11 @@
 
 def barplot(output_dir: str, table: pd.DataFrame, taxonomy: pd.Series,
             metadata: Metadata) -> None:
+    ids_not_in_metadata = set(table.index) - set(metadata.ids)
+    if ids_not_in_metadata:
+        raise ValueError('Feature table contains the following IDs not '
```

Comment:
How about "Feature IDs found in the table are missing in the metadata: foo, bar, baz"? This will match the language used in #109 (a similar ID-check PR).

Reply:
See #127 --- I missed this typo.

cc @Oddant1